index.js

```diff
@@ -141,44 +185,61 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+
+let id = 1
 
-// console.log(takeATicketNumber(line))
+function takeATicketNumber(arr) {
+  arr.push(id)
+  return `Welcome. You are ticket number ${id++}`
+}
+
+console.log(takeATicketNumber(line))
```

Comment:
The logic for this is great! The only thing I'd change is to start your counter from 0 instead of 1.